src/librustc_hir/lang_items.rs

```diff
@@ -154,143 +153,143 @@ pub fn extract(attrs: &[ast::Attribute]) -> Option<(Symbol, Span)> {
+    CharImplItem,                  sym::char,               char_impl,               Target::Impl;
+    StrImplItem,                   sym::str,                str_impl,                Target::Impl;
+    SliceImplItem,                 sym::slice,              slice_impl,              Target::Impl;
+    SliceU8ImplItem,               sym::slice_u8,           slice_u8_impl,           Target::Impl;
+    StrAllocImplItem,              sym::str_alloc,          str_alloc_impl,          Target::Impl;
+    SliceAllocImplItem,            sym::slice_alloc,        slice_alloc_impl,        Target::Impl;
+    SliceU8AllocImplItem,          sym::slice_u8_alloc,     slice_u8_alloc_impl,     Target::Impl;
+    ConstPtrImplItem,              sym::const_ptr,          const_ptr_impl,          Target::Impl;
+    MutPtrImplItem,                sym::mut_ptr,            mut_ptr_impl,            Target::Impl;
+    ConstSlicePtrImplItem,         sym::const_slice_ptr,    const_slice_ptr_impl,    Target::Impl;
+    MutSlicePtrImplItem,           sym::mut_slice_ptr,      mut_slice_ptr_impl,      Target::Impl;
+    I8ImplItem,                    sym::i8,                 i8_impl,                 Target::Impl;
+    I16ImplItem,                   sym::i16,                i16_impl,                Target::Impl;
+    I32ImplItem,                   sym::i32,                i32_impl,                Target::Impl;
+    I64ImplItem,                   sym::i64,                i64_impl,                Target::Impl;
+    I128ImplItem,                  sym::i128,               i128_impl,               Target::Impl;
+    IsizeImplItem,                 sym::isize,              isize_impl,              Target::Impl;
+    U8ImplItem,                    sym::u8,                 u8_impl,                 Target::Impl;
+    U16ImplItem,                   sym::u16,                u16_impl,                Target::Impl;
+    U32ImplItem,                   sym::u32,                u32_impl,                Target::Impl;
+    U64ImplItem,                   sym::u64,                u64_impl,                Target::Impl;
+    U128ImplItem,                  sym::u128,               u128_impl,               Target::Impl;
+    UsizeImplItem,                 sym::usize,              usize_impl,              Target::Impl;
+    F32ImplItem,                   sym::f32,                f32_impl,                Target::Impl;
+    F64ImplItem,                   sym::f64,                f64_impl,                Target::Impl;
+    F32RuntimeImplItem,            sym::f32_runtime,        f32_runtime_impl,        Target::Impl;
+    F64RuntimeImplItem,            sym::f64_runtime,        f64_runtime_impl,        Target::Impl;
+
+    SizedTraitLangItem,            sym::sized,              sized_trait,             Target::Trait;
+    UnsizeTraitLangItem,           sym::unsize,             unsize_trait,            Target::Trait;
+    // trait injected by #[derive(PartialEq)], (i.e.   sym::Partial EQ).
```

Comment:
your search and replace went a bit overboard here ;)

Reply:
Yes! I will fix.